wiring/inc/spark_wiring_logging.h

```diff
@@ -577,14 +577,16 @@ class LogManager {
 
     Vector<LogHandler*> activeHandlers_;
 
+    volatile bool output_active_;
```

Comment:
I know we tend to mix different styles in the code, but let's at least make sure that the code within a single file follows the same naming convention :) Also, `volatile` isn't really necessary here, since the access to this member variable is already guarded by a mutex.

Reply:
I started with the word `active_` and then thought it would be better to say what exactly is active.  It wasn't my intention to mix `snake_case` with `camelCase`.  I'll fix it :)